eslint.config.mjs

```diff
@@ -304,6 +304,12 @@ export default tseslint.config([
           ([ruleName, severity]) => [ruleName, severity === 'off' ? 'off' : 'error'],
         ),
       ),
+      '@eslint-react/no-forbidden-props': [
+        'error',
+        {
+          forbid: ['className', '/_/'],
```

Comment:
Forbidding `className` might fall apart if we use certain libraries that expect that in their types, but this and/or TypeScript should blow up very obviously if we ever hit that case.